### DISCUSSION TITLE
How to enable disable menu points on the ISR Plus's menu?

### DISCUSSION THREAD

**Discussion by GSzabados** [fredlcore/BSB-LAN#402] [Q&A] [answered]
Hi,

I have an ISR Plus to control the heater. I have a few unwanted manu points what happen to be visible in the user menu, like holiday settings for Heating Circuit 3 and 5, what I do not at all. It should list only HC 1, 2 and 4/DHW.

Does anybody know where these menu points can be removed from the user menu of the ISR Plus?

Thanks in advance for the help!

Best Regards,

Gábor Szabados

**Comment by 1coderookie** [comment, marked as answer]
Hi, 
they can't be removed from the ISR itself.

**Reply by GSzabados** [reply]
Thanks for confirming. I swear that I had before HC1,2,3,4(DHW) but not 5 listed until I started to look at some parameters yesterday or a day before. I just thought if it happens to be there then must be a way to remove it as well. Well, then no option for that. :(

**Reply by 1coderookie** [reply]
Well, did you activate HC5 somewhere while you were playing around..?

**Reply by GSzabados** [reply]
I haven't even changed any settings just looked at the current parameters. There is no HC5 listed anywhere else but for holidays.

**Reply by GSzabados** [reply]
Damn, I wrote wrongly, it is not holiday, but time program. 

![20211010_122313](https://user-images.githubusercontent.com/35445496/136691758-e60bb8df-2054-43e2-be40-22f7e1cb51a7.jpg)

**Reply by 1coderookie** [reply]
Not in the menu "configuration" of the ISR maybe? And you didn't set a time for that? Well, then just don't mind.. ;)

**Reply by GSzabados** [reply]
This how in BSB-LAN

![Screenshot_20211010-122507_Chrome](https://user-images.githubusercontent.com/35445496/136691793-c4b82b06-6f4a-4296-a443-94b9c94f0bf9.jpg)

**Reply by 1coderookie** [reply]
Yes, that's what I meant, it's a time program.. And did you set any values there? Click on that category in BSB-LAN..

**Reply by 1coderookie** [reply]
For your other question (in the other thread): Go back to the main view of the QAA and take a picture please and post it there, then we can probably see if summer or winter is active or what's up there..

**Reply by GSzabados** [reply]
I will do once I am back at home. My wife is a bit angry that we are late again...

**Reply by 1coderookie** [reply]
No hurry, wife first - you know what they say - "happy wife, happy life"! ;)

**Reply by GSzabados** [reply]
> And did you set any values there? Click on that category in BSB-LAN..

I haven't set any value, by hitting set. I have looked into the category after that I noticed, but I haven't set anything form BSB-LAN.

**Reply by 1coderookie** [reply]
Ah ok, well then you have to live with that I guess - at least I don't know any way to 'disable' unnused categories/entries within the ISR itself (which btw wouldn't make any sense to me anyway - because then people would probably not find the way to enable that again and therefore they'd miss these options/functions). _Probably_ these entries aren't shown in the 1st enduser-level and they only show up in the 'higher' user-levels? But I'm not sure about that also. 
Anyway: just don't mind them and then you're good.. ;)
